jdk/make/CompileLaunchers.gmk

```diff
@@ -441,6 +446,10 @@ ifeq ($(OPENJDK_TARGET_OS), windows)
       -DJAVA_ARGS='{ "-J-ms8m"$(COMMA) "sun.security.krb5.internal.tools.Ktab"$(COMMA) }'))
 endif
 
+$(eval $(call SetupLauncher,jps, \
+    -DJAVA_ARGS='{ "-J-ms8m"$(COMMA) "openj9.tools.attach.diagnostics.Jps"$(COMMA) }' \
```

Comment:
What is the reason for the `-ms8m`? This seems to be the default, at least on 64-bit.